### PR TITLE
[INS-246 Drew Sukova] Created control-flow.adoc from old book

### PR DESCRIPTION
**Pull request by lsukova**
Overhauled the control flow page. Removed some examples and demonstrations that I felt interfered with the flow of the page, let me know if you're partial to any of the ones I removed. Thanks!

### CODE REVIEW

**Comment by lacykq** [comment]
@lsukova @daniakhanpurdue 

I appreciate all of your work transferring and editing information from the old book to the new one!

I'd love to have you go back to all of your pull requests and review one more time to transform content into plain language. Here's one good [resource](https://www.fda.gov/about-fda/plain-writing-its-law/plain-language-principles) about plain language. Here's another [resource](https://the-examples-book.com/book/how-to-contribute#designing-for-accessibility) that is already in the book as well - it consists more information but for now, writing information in plain language is a good start. 

Please let me know if you have any question

File: content/modules/python/pages/control-flow.adoc [hunk 1]
```diff
@@ -0,0 +1,429 @@
...
+my_odds = []
+for li in my_list:
+    for number in li:
+        if number % 2 == 1:
+            my_odds.append(number)
+        
+print(my_odds)
+----
+
+You won't get any bonus points for using list comprehension in place of nested loops -- so which one is easier to read? Consensus would probably land with the nested loops.
```
Comment (naomim01) [inline content/modules/python/pages/control-flow.adoc:424]: Nested loops are much easier to read than list comprehension.

**Review by naomim01 — APPROVED** [review]

**Review by naomim01 — COMMENTED** [review]
Take a look again at some of the statements/explanations and see if there are easier ways to explain them.

**Comment by dglass19** [comment]
I updated from Naomi's comments and will merge the commit

**Pull request merged**